unity-wrapper/src/main/java/com/linecorp/linesdk/unitywrapper/activity/LineSdkWrapperActivity.kt

```diff
@@ -0,0 +1,150 @@
+                    result.responseCode.ordinal,
+                    "login is canceled."
+                )
+                CallbackPayload(
+                    identifier,
+                    gson.toJson(
+                        errorForUnity
+                    )
+                ).sendMessageError()
+            }
+
+            LineApiResponseCode.SUCCESS -> {
+                Log.d(TAG, "login success")
+                val resultJsonString = gson.toJson(
+                    LoginResultForUnity.convertLineResult(result))
+                CallbackPayload(
+                    identifier,
+                    resultJsonString
+                ).sendMessageOk()
+            }
+
+            else -> {
+                Log.d(TAG, "login error")
+                val errorForUnity = ErrorForUnity(
+                    result.responseCode.ordinal,
+                    result.errorData.message ?: "login error"
+                )
+                CallbackPayload(
+                    identifier,
+                    gson.toJson(
+                        errorForUnity
+                    )
+                )
+                    .sendMessageError()
+            }
+        }
+
+        finish()
+    }
+
+    private fun startLineSdkLoginActivity() {
+        val lineAuthenticationParams = LineAuthenticationParams.Builder().scopes(scope).build()
+        val loginIntent = if (onlyWebLogin) {
+            LineLoginApi.getLoginIntentWithoutLineAppAuth(
+                this,
+                channelId,
+                lineAuthenticationParams
+            )
+        } else {
+            LineLoginApi.getLoginIntent(
+                this,
+                channelId,
+                lineAuthenticationParams
+            )
+        }
+        startActivityForResult(loginIntent,
+            REQUEST_CODE_LOGIN
+        )
+    }
+
+    companion object {
+        private const val KEY_IDENTIFIER = "identifier"
+        private const val KEY_CHANNEL_ID = "channelId"
+        private const val KEY_SCOPE = "scope"
+        private const val KEY_ONLY_WEB_LOGIN = "onlyWebLogin"
+        private const val KEY_BOT_PROMPT = "botPrompt"
+
+        private const val REQUEST_CODE_LOGIN: Int = 1234
+        private const val TAG: String = "LineSdkWrapperActivity"
+        private val gson: Gson = Gson()
+
+        fun startActivity(
+            activity: Activity,
+            identifier: String,
+            channelId: String,
+            scope: String,
+            onlyWebLogin: Boolean,
+            botPrompt: String
+        ) {
+            val intent = Intent(activity, LineSdkWrapperActivity::class.java).apply {
+                addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION)
+                putExtra(KEY_IDENTIFIER, identifier)
+                putExtra(KEY_CHANNEL_ID, channelId)
+                putExtra(KEY_SCOPE, scope)
+                putExtra(KEY_ONLY_WEB_LOGIN, onlyWebLogin)
+                putExtra(KEY_BOT_PROMPT, botPrompt)
```

Comment:
In `parseInent()`, the `KEY_BOT_PROMPT` is not parsed.  Do we need to pass `botPrompt` into the `startActivity`?